readthedocs/doc_builder/environments.py

```diff
@@ -539,6 +542,8 @@ def create_container(self):
                     }
                 }),
                 detach=True,
+                environment = {'READTHEDOCS_VERSION' : self.version.slug,
```

Comment:
Ok, this was the thing that confused me if I should still provide this to the DockerEnvironment.